app/services/doc_auth/mock/result_response.rb

```diff
@@ -34,20 +36,24 @@ def errors
             image_metrics = file_data.dig('image_metrics')
             failed = file_data.dig('failed_alerts')
             passed = file_data.dig('passed_alerts')
-            liveness_result = file_data.dig('liveness_result')
+            face_match_result = file_data.dig('portrait_match_results', 'face_match_result')
             classification_info = file_data.dig('classification_info')
             # Pass and doc type is ok
-            if [doc_auth_result, image_metrics, failed, passed,
-                liveness_result, classification_info].any?(&:present?)
+            if [doc_auth_result, image_metrics, failed, passed, face_match_result,
+                classification_info].any?(&:present?)
```

Comment:
I feel this might be clearer with some line breaks?
```suggestion
            has_fields = [
              doc_auth_result,
              image_metrics,
              failed,
              passed,
              face_match_result,
              classification_info
            ].any?(&:present?)
            
            if has_fields
```

Reply:
You're right, that is easier to read through.